fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -161,6 +162,7 @@ def test_create_big_object_and_download_it_from_container(self):
         suffix = datetime.utcnow().strftime('%Y%m%d%H%M%S%m')
         containerName = TEST_CONTAINER_PREFIX + suffix
         big_object_name = TEST_BIG_OBJECT_PREFIX + suffix + TEST_BIG_FILE_EXTENSION
+        remote_big_object_name = TEST_BIG_OBJECT_REMOTE_PREFIX + TEST_BIG_OBJECT_PREFIX + suffix + TEST_BIG_FILE_EXTENSION
```

Comment:
Improbable, but what would happen if two test of different regions execute this line in the same millisecond ?
What do you think if we also add the region name to the local filename?